pkg/vm/vm_test.go

```diff
@@ -481,6 +481,158 @@ func TestINVERTgood3(t *testing.T) {
 	assert.Equal(t, int64(-0x6A), vm.estack.Peek(0).BigInt().Int64())
 }
 
+func TestCATbadnoargs(t *testing.T) {
+	prog := makeProgram(CAT)
+	vm := load(prog)
+	vm.Run()
+	assert.Equal(t, true, vm.state.HasFlag(faultState))
+}
+
+func TestCATbadonearg(t *testing.T) {
```

Comment:
nit:
What about having camel case naming:
TestCATBadOneArg or smth like TestCAT_BadOneArg

btw I see that we're using bad/good terms for naming positive and negative cases. Dunno if it's the right choice but would like to mention it in some guide so we won't have different naming convention in tests

Reply:
`TestCAT_BadOneArg` is not goesque enough, but the first one is actually nice, it's just that I was a bit lazy with it. Will fix.

Reply:
Fixed, please check.